src/control/system/membership.go

```diff
@@ -444,9 +444,8 @@ func (m *Membership) MarkRankDead(rank Rank) error {
 	}
 
 	if member.State().isTransitionIllegal(MemberStateEvicted) {
-		m.log.Debugf("skipping illegal member state update for rank %d: %s->%s",
+		return errors.Errorf("llegal member state update for rank %d: %s->%s",
```

Comment:
Minor typo in the error message, should probably fix that before landing

Reply:
fixing in https://github.com/daos-stack/daos/pull/4875